packages/commons/tests/utils/cloudformation-deployments.ts

```diff
@@ -0,0 +1,223 @@
+// This file is duplicating CloudFormationDeployments define in https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/lib/api/cloudformation-deployments.ts
+// This was needed to overwrite isAssetManifestArtifact function which was preventing proper assets to be created due to difference between cloudAssembly in @aws-cdk vs. aws-cdk-lib
+
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/member-ordering */
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
+/* eslint-disable @typescript-eslint/member-delimiter-style */
+/* eslint-disable newline-before-return */
+/* eslint-disable func-style */
```

Comment:
I went through these rules and I think they are fine as it is for global level.

This file is only exceptional since it's duplicated from an other project. I prefer to just disable lint in this file and keep it exactly as it is for ease of maintenance.